src/databudgie/adapter/base.py

```diff
@@ -58,9 +59,8 @@ def query_database(session: Session, query: str) -> Generator[List[Any], None, N
             columns: List[str] = list(cursor.keys())
             yield columns
 
-            row: List[Any]
             for row in cursor:
-                yield row
+                yield list(row)
```

Comment:
why did this change

Reply:
another mypy typing thing. `row` is not a list, and there's not really a good cross-sqlalchemy-version alternative to type it as either. And something more python-native-generic causes other issues at the use-site. So i figured explicitly casting it was the simplest/least invasive change